### PR TITLE
Docker file for ansible

### PR DESCRIPTION
**Pull request by ptolemy428**
## Approvals

- [x] verified
- [x] Tim
- [ ] Angel

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:14.04
```
Comment (ptolemy428) [inline Dockerfile]: I was thinking abt using the new Alphine linux as the base for our images (Docker are switching its base images to Alphine linux), but decided not to do so for a couple of reasons:
1. To take advantage of existing deb repos. The latest ansible has been distributed as deb packages which can be installed with all dependencies (very smoothly) onto a bare-bone ubuntu image. The latest apk pkg for ansible is still 1.9.4

2. Given this is a tool image (not an app image), it is better to use a more popular distribution which makes troubleshooting and installing other packages easier in the future.

3. The final image size is about 250MB. The Alpine based will be around 100MB with all dependencies installed, but the difference is not that big.

**Comment by ptolemy428** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;rmWie2Qr&#x2F;132-create-docker-container-to-run-ansible-to-configure-ec2-instance-with-test-config"> Create Docker container to run Ansible to configure ec2 instance with test config</a>

**Comment by TimMoore** [comment]
Looks good to me. Will we also need `boto` installed?

**Comment by ptolemy428** [comment]
At this stage, the ansible playbooks for baking AMIs do not have dependencies on boto. we have a separate card to make the binary repo (s3 backed) accessible internally. We can add boto in at the stage if we decide to download binaries using boto rather than get_url method.

**Comment by hebbo** [comment]
:+1:

**Pull request merged**